readme.md

```diff
@@ -40,7 +40,40 @@ importance (although feel free to ask them to try to learn more about _why_
 12. Note, in order to be able to perform these responsibilities it's important
 that POs have been given the authority and means necessary to complete them. :muscle:
 
-* product backlog: a prioritised features list for the product found on github.
+*product backlog: a prioritised features list for the product found on github.
+
+## What is the backlog?
+
+The backlog is your project's to do list, a list of all the features and things
```

Comment:
This definition is a _good_ starting point, however it may need some expansion/clarification.
quite often there will be items in the backlog that are _not_ "Todo List" 
but rather "ideas" or "feature requests" which have not yet been validated or prioritised 
and therefore are _not_ "todo". 
If people in the team _assume_ that everything in the backlog is "todo" 
people can end up squandering time building the "wrong thing" ...
Provided everyone in the team is clear that only issues/stories that have been validated and prioritised are considered "ready for dev", then it's fine.

Reply:
Good point